src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -159,4 +159,58 @@
         <Setter Property="Width" Value="{StaticResource StandardBoxMinWidth}" />
         <Setter Property="HorizontalAlignment" Value="Left" />
     </Style>
+    <!-- DELETE BUTTON STYLING IN COLORSCHEMES.XAML FILE -->
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Light">
+            <Style x:Key="SecondaryTextBlockStyle" TargetType="TextBlock">
+                <Setter Property="Foreground" Value="{ThemeResource SystemBaseMediumColor"/>
+                <SolidColorBrush x:Key="ButtonBackground"
+                                    Color="Firebrick" />
+                <SolidColorBrush x:Key="ButtonBackgroundPointerOver"
+                                    Color="#C23232" />
+                <SolidColorBrush x:Key="ButtonBackgroundPressed"
+                                    Color="#A21212" />
+                <SolidColorBrush x:Key="ButtonForeground"
+                                    Color="White" />
+                <SolidColorBrush x:Key="ButtonForegroundPointerOver"
+                                    Color="White" />
+                <SolidColorBrush x:Key="ButtonForegroundPressed"
+                                    Color="White" />

```

Comment:
All of these likely need to turn into `Setter` items on `Properties` that are a part of `Button` with the color or brush binding to the relevant properties. It's sort of a backwards declaration to how it is done on an individual button override.

Reply:
> It doesn't look like this builds either. Did you F5 it in Visual Studio to see if your shared style works?

Thanks for reviewing.
I have gone through all your replies and I'll make all the required changes.
I'll F5 it and see if it works after correcting the current code.